src/test/java/com/sda/project/management/service/SprintServiceTest.java

```diff
@@ -39,4 +39,21 @@ void shouldAddTaskToSprint() {
         // then
         assertThat(tasks).hasSize(2);
     }
+
+    @Test
+    void shouldCreateSprint(){
+
+        Sprint sprint = new Sprint();
+        sprint.setName("alex");
+        sprint.setDateFrom(LocalDate.now());
+        sprint.setDateTo(LocalDate.now().plusDays(5));
+        sprint.setStoryPoints(2);
+        sprint.setSprintGoal("terminate");
+
+        sprintService.save(sprint);
+
+        String test = sprintService.findById(sprint.getId()).getSprintGoal();
+
+        assertThat(test).isEqualTo("terminate");
```

Comment:
no point in checking a specific field, just cause you had to add that field as part of a task. assert not null is fine